pkg/host-timesync-daemon/main.go

```diff
@@ -51,10 +51,10 @@ func rtcReadTime() rtcTime {
 	defer f.Close()
 	result := rtcTime{}
 	arg := uintptr(0)
-	arg |= (iocREAD << iocDIRSHIFT)
-	arg |= (rtcRDTIMETYPE << iocTYPESHIFT)
-	arg |= (rtcRDTIMENR << iocNRSHIFT)
-	arg |= (unsafe.Sizeof(result) << iocSIZESHIFT)
```

Comment:
unless lint or vet complains about these, I prefer to leave them with the `(   )`. Operation precedence in go should do bitwise shift before bitwise OR and especially before assignment, but it can be confusing.

Reply:
I'll remove those changes for now

Reply:
Just watch. As soon as I enable golangci-lint it will complain 😆 

We will cross that bridge when we come to it.

Reply:
🥇